.ci/test-install-docs.sh

```diff
@@ -17,7 +17,11 @@ info()
 # commands it specified result in a working system.
 check_install_guides()
 {
-	[ -n "$TRAVIS" ] && info "Not testing install guide as Travis lacks modern distro support and VT-x" && return 
+	if [ -n "$TRAVIS" ]
+	then
```

Comment:
We don't prescribe a coding style atm. However, we could if anyone wants to work on a new doc (and fight all the fights to get it landed :), or maybe it could be an update to:

- https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md